service/server.py

```diff
@@ -239,6 +250,10 @@ def _show_full_title_data():
     return app.config.get('SHOW_FULL_TITLE_DATA')
 
 
+def _show_full_title_pdf():
```

Comment:
I'm such a hypocrite, you just followed my convention. What do you think the names should be?